### ISSUE TITLE
Upgrade to Bond 7.0.0 : Ambiguous reference to member 'bind(to:)'

### ISSUE THREAD

**Issue by npvisual** [DeclarativeHub/Bond#580]
### Context
Bond "7.0.0"
ReactiveKit "v3.9.7"
Differ "1.3.0"

### Problem Description
Before I upgraded to Bond 7.0.0 I was able to bind a simple Signal<[Int], NoError> to a MutableObservableArray of the same type (Int is just used here as an example).

After addressing all the other migration issues, this is the only one I have left which I can't seem to be able to resolve. The compiler complains about : 

```
Ambiguous reference to member 'bind(to:)'

1. Found this candidate (ReactiveKit.SignalProtocol)
2. Found this candidate (ReactiveKit.SignalProtocol)
```

For example : 

```
    public let testProp : Property<[Int]> = Property([0])
    public let testArray : MutableObservableArray<Int> = MutableObservableArray([])
...
        testProp
            .toSignal()
            .bind(to: testArray)
```
will trigger that complaint from the compiler, whereas it didn't use to with previous version of Bond.

Any ideas on how to fix this ?

Thanks !

**Comment by srdanrasic** [comment]
Implemented in 7.2.0.

**Comment by npvisual** [comment]
Works great in 7.2.0. Thanks !
